lib/config/index.js

```diff
@@ -1,7 +1,25 @@
 var nconf = require('nconf'),
-    path = require('path');
+    fs = require('fs'),
+    path = require('path'),
+    findRoot = require('find-root'),
+    defaults = {},
+    getParentPath = function (parent) {
+        if (!parent  || !parent.filename) {
+            return null;
+        }
 
-var defaults = require(path.join(process.cwd(), 'config.example'));
+        if (parent.filename.match(/ghost-ignition/gi)) {
+            return getParentPath(parent.parent);
+        }
+
+        // finds the root with package.json based on a path
+        return findRoot(parent.filename);
+    },
+    parentPath = getParentPath(module.parent);
+
+if (parentPath && fs.existsSync(path.join(parentPath, 'config.example.js'))) {
```

Comment:
This should be `config.example.json`. Ignition/nconf expects `.json` files. The fact that this picked up `config.example.js` from Ghost and worked is because we were doing a `require` of just `config.example`.